channels/us.m3u

```diff
@@ -6069,8 +6069,8 @@ https://tdameritrade-distro.amagi.tv/playlist.m3u8
 https://eu-nl-012.worldcast.tv/dancetelevisionthree/dancetelevisionthree.m3u8
 #EXTINF:-1 tvg-id="TELEMUNDO.us" tvg-name="TELEMUNDO" tvg-country="US" tvg-language="" tvg-logo="https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcQezVfVamzd08ywLqOjqVqfOtEL-ZyphG2b0w&usqp=CAU" group-title="",TELEMUNDO
 https://wmaqlive-f.akamaihd.net/i/wmaqb1_1@24420/index_1286_av-p.m3u8?sd=10&rebase=on
-#EXTINF:-1 tvg-id="KTAZ.us" tvg-name="Telemundo 40 [KTAZ] (Phoenix | AZ)" tvg-country="US" tvg-language="Spanish" tvg-logo="https://i.imgur.com/pb0b7A4.png" group-title="General",Telemundo 40 Arizona (720p)
-https://ktazlive-lh.akamaihd.net/i/KTAZ_LIVE1@183609/master.m3u8
+#EXTINF:-1 tvg-id="KTAZ.us" tvg-name="Telemundo 40 [KTAZ] (Phoenix | AZ)" tvg-country="US" tvg-language="Spanish" tvg-logo="https://i.imgur.com/pb0b7A4.png" group-title="General",Telemundo 40 Arizona
+https://streams.the6tv.duckdns.org:2443/locals/Pheonix/ktaz-dt1.m3u8
```

Comment:
Is it supposed to be Pheonix?

Reply:
https://fr.wikipedia.org/wiki/Phoenix_(Arizona)